.github/workflows/gh-pages.yaml

```diff
@@ -80,6 +80,7 @@ jobs:
           publish_dir: build/web
           keep_files: true
           destination_dir: "${{ github.event.pull_request.number }}"
+          force_orphan: true
```

Comment:
How this is related to the sdk upgrade? It should have its own commit with the "why explanation"